packages/css-library/src/stylesheets/uswds-typography.scss

```diff
@@ -0,0 +1,38 @@
+@forward 'uswds-theme';
+
+@import 'uswds-core/src/styles/mixins/typography/usa-content-styles';
+@import 'uswds-core/src/styles/mixins/typography/typeset';
+
+/**
+  Normalization for Source Sans Pro typeface.
+
+  Typefaces vary in optical size. This means that at any 
+  specific pixel  value, an optically small typeface like  
+  Source Sans Pro will appear smaller than an optically 
+  large typeface like Merriweather.
+
+  The `typeset()` mixin will calculate the appropriate 
+  font-size for  a given typeface. The `ui` value will 
+  use the default value for $theme-body-font-family.
+
+  font-family: Source Sans Pro Web, Helvetica Neue, Helvetica, Roboto, Arial, sans-serif;
+  font-size: 1.06rem;
+  line-height: 1.5;
+
+  https://designsystem.digital.gov/design-tokens/typesetting/overview/
+*/
+p, ol, ul, li {
```

Comment:
These elements `p, ol, ul, li` seemed to cover the bulk of page content when I was spot checking vets-website locally. But we will want to verify this in a review instances and make adjusts if needed.

Reply:
A couple of other tags that come to mind are `label` and `legend`. Do you think those are worth adding to the list?

Reply:
@micahchiang good thinking. I'll double check those elements. I think forms library is already transitioned to mostly using USWDS web components so those forms should already be loading with 16.96px but we should account for non-forms library stuff.

Reply:
@micahchiang I'm seeing `legend` style is applied through (at the moment) [USWDS v1 styles here](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/css-library/src/stylesheets/formation-overrides/elements/_inputs.scss#L163-L166) with its own specific font-size and the `rem` value already converted to the 16px root font size. There are other form styles as well that you might be thinking of. 

I'm not sure if we want to start applying that `typeset` mixin to those elements just yet but maybe instead at some point replace those v1 styles with v3 imports specifically related to legend, label, ect if they're available?